packages/@aws-cdk/aws-cognito/test/user-pool.test.ts

```diff
@@ -1271,6 +1271,27 @@ describe('User Pool', () => {
       })).toThrow(/enableSmsRole cannot be disabled/);
     });
   });
+
+  test('email transmission with cyrillic characters are encoded', () => {
+    // GIVEN
+    const stack = new Stack();
+
+    // WHEN
+    new UserPool(stack, 'Pool', {
+      emailSettings: {
+        from: 'from@домен.рф',
```

Comment:
What about the parts before the `@` sign? 

If they contain unicode characters, fairly sure they should NOT be converted to punycode.

Reply:
Good catch!
Will confine the encoding to the domain-part only 👍